### ISSUE TITLE
Add documentation for Envoy as reverse proxy

### ISSUE THREAD

**Issue by phlax** [solid/solidproject.org#414]
**Search terms you've used**

envoy, proxy

**What problem are you trying to solve? Please describe.**

offer to contribute documentation.

if you are interested to include documentation for Envoy, i would be happy to create a pull request

**What existing documentation is relevant to your problem, and why is it insufficient?**

there are currently guides to using Nginx and Apache, it would be great if there were Envoy docs also.

* If you have them provide pointers to relevant related information resources.

Envoy is an increasingly popular FLOSS solution for proxying.

Project is here

https://github.com/envoyproxy/envoy

**Comment by Vinnl** [comment]
@solid/editors this sounds like a fine addition, no?

**Comment by justinwb** [comment]
Absolutely. @phlax if you're willing to submit documentation we'd be happy to review and add!

**Comment by phlax** [comment]
cool - ill put something together in the next couple of days, i have it ~working locally
